content/applications/finance/accounting/reporting.rst

```diff
@@ -24,8 +24,11 @@ the details.
 .. image:: reporting/reporting-annotate.png
    :alt: Annotate reports.
 
-To export reports in PDF or XLSX format, click :guilabel:`PDF` or :guilabel:`XLSX` at the top of the
-page.
+To export reports in PDF or XLSX format:
+
+- click :guilabel:`PDF` at the top of the page
+- click the :icon:`fa-caret-down` (:guilabel:`down arrow`) next to the :guilabel:`PDF` button, and
+  select :guilabel:`XLSX`.
```

Comment:
I think this could all go in one sentence since it's quite short?
```suggestion
To export reports in PDF or XLSX format, click :guilabel:`PDF` at the top or click the :icon:`fa-caret-down` (:guilabel:`down arrow`) icon next to the :guilabel:`PDF` button and select :guilabel:`XLSX`.
```

Reply:
@auva-odoo Yes, sure, I hesitated :) Thanks!